sdcm/logcollector.py

```diff
@@ -281,6 +281,31 @@ def get_grafana_annotations(self, node):  # pylint: disable=inconsistent-return-
             LOGGER.warning("unable to get grafana annotations [%s]", str(ex))
             return ""
 
+    @staticmethod
+    def dashboard_exists(node: CollectingNode, uid: str) -> bool:
```

Comment:
kudos for using hinting !!!

Reply:
Like for type annotations!

Reply:
but, we can't backport to 3.2/3.1 since the branches are not running using Python 3

Reply:
we could change it on backport, but we need to pay close attention while doing so.
i.e. I think for anything that needs backport, lets not use python3 only features just yet.

Reply:
fixed for backport